src/game/behaviors/pokey.inc.c

```diff
@@ -143,20 +133,19 @@ void bhv_pokey_body_part_update(void) {
 }
 
 /**
- * When mario gets within range, spawn the 5 body parts and enter the wander
- * action.
+ * When mario gets within range, spawn the POKEY_NUM_SEGMENTS body parts and enter the wander action.
  */
 static void pokey_act_uninitialized(void) {
     struct Object *bodyPart;
     s32 i;
     s16 partModel;
 
-    if (o->oDistanceToMario < 2000.0f) {
+    if (o->oDistanceToMario < 4000.0f) {
```

Comment:
why? @Arceveti

Reply:
Oh whoops that was just increased for testing. It should probably use o->oDrawingDistance anyways.

Reply:
@Arceveti can you revert to 2000

Reply:
I've already changed it to use o->oDrawingDistance in dev